src/frontend/hooks/useHentTilbakekrevingsbehandlinger.ts

```diff
@@ -4,12 +4,14 @@ import { useHttp } from '@navikt/familie-http';
 
 import { hentTilbakekrevingsbehandlinger } from '../api/hentTilbakekrevingsbehandlinger';
 
-export const TILBAKEKREVINGSBEHANDLINGER_QUERY_KEY_PREFIX = 'tilbakekrevingsbehandlinger';
+export const HentTilbakekrevingsbehandlingerQueryKeyFactory = {
+    fagsak: (fagsakId: number | undefined) => ['tilbakekrevingsbehandlinger', fagsakId],
```

Comment:
Trenger egentlig denne å ta i mot `undefined`?

```suggestion
    fagsak: (fagsakId: number) => ['tilbakekrevingsbehandlinger', fagsakId],
```